backbone/src/main/java/org/researchstack/backbone/answerformat/EmailAnswerFormat.java

```diff
@@ -11,6 +13,7 @@
     public EmailAnswerFormat()
     {
         super(MAX_EMAIL_LENGTH);
+        setInputType(InputType.TYPE_CLASS_TEXT | InputType.TYPE_TEXT_VARIATION_EMAIL_ADDRESS);
```

Comment:
Thought it was odd, and wanted to call it out for you guys, but you can't just have InputType.TYPE_TEXT_VARIATION_EMAIL_ADDRESS without also having InputType.TYPE_CLASS_TEXT. Same goes for any of the variations of TYPE_NUMBER